src/cockpit/389-console/src/lib/plugins/attributeUniqueness.jsx

```diff
@@ -611,7 +611,8 @@ class AttributeUniqueness extends React.Component {
                     console.info("attrUniqOperation", "Result", content);
                     this.props.addNotification(
                         "success",
-                        cockpit.format(_("The $0 operation was successfully done on \"$1\" entry"), action, configName)
+                        cockpit.format("The $0 operation was successfully done on \"$1\" entry, please restart the instance for these changes to take effect.",
```

Comment:
earlier it was in `_()`: Message is hardcoded and won't be translated, not sure if it will break anything!

Reply:
It only translates it if there is a corresponding mapping in ja.po.  It won't match anymore, so by removing the "function" it is easier to spot as a place where we need a new language mapping.